drivers/lhal/src/bflb_gpio.c

```diff
@@ -26,16 +29,31 @@ void bflb_gpio_init(struct bflb_device_s *dev, uint8_t pin, uint32_t cfgset)
 #endif
 
 #if defined(BL702) || defined(BL602) || defined(BL702L)
```

Comment:
Only for bl702

Reply:
BL602 and BL702L have no such register.

Reply:
There is no documentation for this register in BL702 RM also. ;-)
I'll correct this and add ifdef to skip this register checking for BL602/BL702L.